screens/Home/MyVcs/GetIdInputModal.tsx

```diff
@@ -28,70 +28,69 @@ export const GetIdInputModal: React.FC<GetIdInputModalProps> = props => {
       headerTitle={t('header')}
       headerElevation={2}>
       <KeyboardAvoidingView
-        style={{flex: 1}}
+        style={Theme.Styles.keyboardAvoidStyle}
         behavior={isIOS() ? 'padding' : 'height'}>
-        <Column fill align="space-between" padding="32 24">
```

Comment:
Can you please fix styling as I am unable to identify the difference here?

Reply:
As it is intermittent issue I was not able to replicate that, but as of my research I found that it may arise due to inappropriate padding and also I was assuming that <Column fill> that we are using may overriding the KeyboardAvoidingView functionality as <Column fill>  will occupies full height. So, I have increased padding and removed <Column fill>